### ISSUE TITLE
CovNet example

### ISSUE THREAD

**Issue by dmccloskey** [dmccloskey/EvoNet#59]
Example Architecture
- https://github.com/pytorch/examples/blob/master/mnist/main.py
- Conv (1 image, 10 depth, 5x5 filter, stride 1), MaxPool (2x2, stride 1), FC (ReLU), Conv (10 images, 2 depth, 5x5 filter, stride 1), MaxPool (2x2, stride 1), FC (ReLU), FC (ReLU, 320 to 50), FC (ReLU, 50 to 10), SoftMax

Objectives:
- [x] Convolution + pooling
- [ ] Transpose convolution
- [ ] Dialation

References:
- https://github.com/vdumoulin/conv_arithmetic/blob/master/README.md
